### PR TITLE
added new test class

### PR DESCRIPTION
**Pull request by PandeyMohit20**
please change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Drum-kits
-# MOHIT KUMAR PANDEY
+
```
Comment (Abyz7667) [inline README.md:2]: # Drumkit

Comment (PandeyMohit20) [inline README.md:2]: #DRUM

**Review by Abyz7667 — COMMENTED** [review]
DONE

**Review by PandeyMohit20 — COMMENTED** [review]
OK

**Pull request merged**